nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/audit/ProcessorAuditor.java

```diff
@@ -379,6 +380,9 @@ public Action generateAuditRecord(ProcessorNode processor, Operation operation,
             if (newConfig.getExecutionNode() != null) {
                 values.put(EXECUTION_NODE, processor.getExecutionNode().name());
             }
+            if (newConfig.isExecutionNodeRestricted() != null) {
```

Comment:
I don't believe this is needed here - it's not a configurable value. It's hardcoded by the developer.